build.sbt

```diff
@@ -1,126 +1,143 @@
-import PlayCrossCompilation.{dependencies, version}
-import play.core.PlayVersion
 import sbt.Path.relativeTo
 
-lazy val majorVersionNumber = 0
+ThisBuild / majorVersion := 1
+ThisBuild / isPublicArtefact := true
 
-lazy val lib = (project in file("."))
+val scala2_12 = "2.12.18"
+val scala2_13 = "2.13.12"
+
+def copySources(module: Project) = Seq(
+  Compile / scalaSource := (module / Compile / scalaSource).value,
+  Compile / resourceDirectory := (module / Compile / resourceDirectory).value,
+  Test / scalaSource := (module / Test / scalaSource).value,
+  Test / resourceDirectory := (module / Test / resourceDirectory).value
+)
+
+def copyPlayResources(module: Project) = Seq(
+  Compile / routes / sources ++= {
+    // compile any routes files in the root named "routes" or "*.routes"
+    val dirs = (module / Compile / unmanagedResourceDirectories).value
+    (dirs * "routes").get ++ (dirs * "*.routes").get
+  }
+)
```

Comment:
@JoPintoPaul do we need the routes stuff ? For play-frontend-hmrc and play-language we did because they both have routers but play-nunjucks doesn't have any compiled routers

Reply:
@oscarduignan LET'S FIND OUT *moves fast and breaks things*